GPUSimpleVector.h

```diff
@@ -9,14 +9,18 @@
 #if defined DIGI_CUDA
 #include <cuda.h>
 #elif defined DIGI_ALPAKA
+#if defined ALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED || defined ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLED || defined ALPAKA_ACC_CPU_B_TBB_T_SEQ_ENABLED || defined ALPAKA_ACC_GPU_CUDA_ENABLED
```

Comment:
Is there a case where `DIGI_ALPAKA` is defined, but none of the four `ALPAKA_ACC_*` is not?